resources/execution_endtoend_script_tags.json

```diff
@@ -3,57 +3,81 @@
     {
       "name":"nop",
       "docker_tag":"nop",
-      "docker_repository":"fogbow/inputdownloader"
+      "docker_repository":"fogbow/inputdownloader",
+      "memory_usage":"16G",
+      "cpu_usage":"1000m"
     },{
       "name":"nop-fake-files",
       "docker_tag":"nop-fake-files",
-      "docker_repository":"fogbow/inputdownloader"
+      "docker_repository":"fogbow/inputdownloader",
+      "memory_usage":"16G",
+      "cpu_usage":"1000m"
     },{
       "name":"nop-download-files",
       "docker_tag":"nop-download-files",
-      "docker_repository":"fogbow/inputdownloader"
+      "docker_repository":"fogbow/inputdownloader",
+      "memory_usage":"16G",
+      "cpu_usage":"1000m"
```

Comment:
what is this `m` in the cpu_usage?

Reply:
1000m = 1 vcpu, this is how kubernetes works with the requirements

Reply:
ok, thanks